packages/design-system/src/components/Accordion/AccordionItem.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { FunctionComponent, useState } from 'react';
+import classNames from 'classnames';
+import uniqueId from 'lodash.uniqueid';
+
+interface AccordionItemProps {
+  children?: React.ReactNode;
+  /**
+   * Class to be applied to the heading `<h2>` tag of an accordion item.
+   */
+  contentClassName?: string;
+  /**
+   * Class to be applied to the content `<div>` tag of an accordion item.
+   */
+  headingClassName?: string;
+  /**
+   * Boolean to expand the accordion.
+   */
+  expanded?: boolean;
+  /**
+   * Text for the accordion item heading.
+   */
+  heading: React.ReactNode | string;
+  /**
+   *  If not provided, a unique id will be automatically generated and used.
+   */
+  id?: string;
+}
+
+const AccordionItem: FunctionComponent<AccordionItemProps> = ({
+  children,
+  contentClassName,
+  headingClassName,
+  expanded,
+  heading,
+  id,
+}) => {
+  // Set the state for opening and closing an accordion item
+  const [open, setOpen] = useState(!!expanded);
+  const onClick = () => {
+    setOpen(!open);
+  };
+  const contentClasses = classNames('ds-c-accordion__content', contentClassName);
```

Comment:
Since `children` is being passed through to the accordion content, can we just name this `className` instead of `contentClassName`?

Reply:
I think we want to have a unique className for the individual content area div as well as the div container that wraps all the accordions.